client/src/components/Feed/PostTabCard.js

```diff
@@ -0,0 +1,144 @@
+import React, { useState, useMemo } from "react";
+import { useSelector } from "react-redux";
+import { Tabs } from "antd-mobile";
+import { Button } from "antd";
+import PostDropdownButton from "components/Feed/PostDropdownButton";
+import CreateReport from "components/CreateReport/CreateReport";
+import { formatDate } from "./utils";
+import { isAuthorOrg, isAuthorUser } from "utils/userInfo";
+import { selectActorId } from "reducers/session/selectors";
+import {
+  StyledPostTabCard,
+  StyledCardHeader,
+  StyledCardBody,
+  StyledUpdateAt,
+  StyledTitlePostCard,
+  StyledDescription,
+  LineBreak,
+  Container,
+  StyledCardFooter,
+  StyledDropDown,
+} from "./StyledPostTabCard";
+
+const CONTENT_LENGTH = 56;
+
+const PostContent = ({
+  posts,
+  user,
+  maxPosts,
+  handlePostDelete,
+  isAuthenticated,
+  fromPage,
+}) => {
+  const [hiddenPosts, setHiddenPosts] = useState(
+    JSON.parse(localStorage.getItem("hiddenPosts")) || {},
+  );
+  const [callReport, setCallReport] = useState(false);
+  const actorId = useSelector(selectActorId);
+
+  const hidePost = (postId) => {
+    localStorage.setItem(
+      "hiddenPosts",
+      JSON.stringify({ ...hiddenPosts, [postId]: true }),
+    ); // objects are fast, better than looking for postId in an Array
+    setHiddenPosts({ ...hiddenPosts, [postId]: true });
+  };
+
+  const content = useMemo(
+    () =>
+      posts
+        .filter((post) => !hiddenPosts[post._id])
+        .slice(0, maxPosts)
+        .map((post) => {
+          let finalContent = post.content;
+          if (finalContent.length > CONTENT_LENGTH) {
```

Comment:
Why are you merging to staging from this branch, instead of from the feature/profile-redesign feature branch?